inkscape/src/main/assembly/dist/lace_ground.inx

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
 
-	<_name>Lace Ground</_name>
+	<_name>Ground from Template</_name>

```

Comment:
Might be confusing with my templates. I suppose you mean your generated matrices. While my templates are InkScape files. How do we disambiguate?